src/components/table/base-table/base-table.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { CellMeasurer, CellMeasurerCache, MultiGrid } from 'react-virtualized';
-import sortBy from 'lodash/sortBy';
```

Comment:
I thought this way ensures smaller bundle size 🤔

Reply:
Well we use `lodash-es` for our ESM bundle which ensures smaller bundle size. 

Consumers who use the cjs bundle will have a larger bundle-size, but we only use that in test environment...

Reply:
Plus doing this doesn't work with the lodash-es switcher

Reply:
that's not a bad idea actually.

Reply:
Okay I tried both approaches, they result in same bundle sizes for consumer and for UI-Kit.

Reply:
thank you for going the extra mile 🙏

Reply:
I want to make another change first though so we can have a better idea of what's happening.